backend/src/services/v2/model.ts

```diff
@@ -19,7 +19,7 @@ import { BadReq, Forbidden, NotFound } from '../../utils/v2/error.js'
 import { convertStringToId } from '../../utils/v2/id.js'
 import { findSchemaById } from './schema.js'
 
-export type CreateModelParams = Pick<ModelInterface, 'name' | 'description' | 'visibility'>
+export type CreateModelParams = Pick<ModelInterface, 'name' | 'team' | 'description' | 'visibility'>
```

Comment:
Should the backend do validation to ensure that the team exists?

Reply:
Decided to leave this for when teams have been fully implemented in the backend